core/templates/dev/head/components/rubrics-editor/rubrics-editor.directive.html

```diff
@@ -1,50 +1,28 @@
-<span>Select Difficulty:</span>
-<div class="dropdown">
-  <button class="btn btn-default dropdown-toggle protractor-test-select-rurbic-difficulty-dropdown"
-          type="button"
-          data-toggle="dropdown"
-          ng-disabled="$ctrl.explanationEditorIsOpen">
-    <[$ctrl.getRubrics()[$ctrl.activeRubricIndex].getDifficulty()]>
-    <span class="caret"></span>
-  </button>
-  <ul class="dropdown-menu">
-    <li class="dropdown-option protractor-test-rubric-difficulty" ng-repeat="rubric in $ctrl.getRubrics()"
-        ng-click="$ctrl.setActiveDifficultyIndex($index)">
-        <[rubric.getDifficulty()]>
-    </li>
-  </ul>
-</div>
-<ul class="nav oppia-option-list nav-stacked nav-pills"
-    role="tablist" ng-model="$ctrl.skillDifficulties">
-  <li ng-repeat="difficulty in $ctrl.skillDifficulties"
-      class="oppia-rule-block oppia-prevent-selection"
-      style="padding-bottom: 2vh;">
-  </li>
-</ul>
-<div class="rubric-editor">
-  <div ng-if="!$ctrl.explanationEditorIsOpen"
+<div ng-repeat="rubric in $ctrl.getRubrics()" class="rubric-editor">
+  <label> <[rubric.getDifficulty()]> </label>
+  <div ng-if="!$ctrl.explanationEditorIsOpen[$index]"
        style="height: 100%; padding-top: 1vh;">
     <div class="oppia-readonly-rule-title"
          ng-class="{'oppia-editable-section': $ctrl.isEditable()}">
       <div class="oppia-rule-preview">
-        <div class="oppia-click-to-start-editing" ng-click="$ctrl.openExplanationEditor()">
+        <div class="oppia-click-to-start-editing" ng-click="$ctrl.openExplanationEditor($index)">
           <i ng-if="$ctrl.isEditable()"
              class="material-icons oppia-editor-edit-icon float-right protractor-test-edit-rubric-explanation"
              title="Edit Rubric Explanation">&#xE254;
           </i>
         </div>
-        <span ng-if="$ctrl.getRubrics()[$ctrl.activeRubricIndex].getExplanation() === ''"
+        <span ng-if="$ctrl.getRubrics()[$index].getExplanation() === ''"
               class="oppia-placeholder">
-          Click here to add rubric for given difficulty
+          The skill description will be autofilled here
```

Comment:
Wait, this seems like an odd message to show to the user. Shouldn't we just autofill it?

Reply:
We do autofill it and it happens live, but when the skill description is empty, this explanation is empty, so added this mesage to let the user know this field will be filled with the description.

Reply:
OK, let's leave it as is for now then. Thanks for explaining!